src/Microsoft.Security.Utilities.Core/UnclassifiedPotentialSecurityKeys/SEC101_061_LooseOAuth2BearerToken.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Security.Utilities
+{
+    public class OAuth2BearerToken : RegexPattern
+    {
+        public OAuth2BearerToken()
+        {
+            Id = "SEC101/061";
+            Name = nameof(OAuth2BearerToken);
+            DetectionMetadata = DetectionMetadata.LowConfidence;
+
+            // https://datatracker.ietf.org/doc/html/rfc6750#section-2.1
+            Pattern = $"(?i)authorization:(\\s|%20)bearer(\\s|%20)<?<refine>[0-9a-z][{WellKnownRegexPatterns.UrlUnreserved}+/=]*)[^[{WellKnownRegexPatterns.UrlUnreserved}+/=]";
+            Signatures = new HashSet<string>(new[] { "sig=", "ret=" });
+        }
```

Comment:
This is added in .NET code but seems not added to `UnclassifiedPotentialSecurityKeys.json`

Reply:
I'm thinking there could be some cases that BearerToken is used without the prefix `authorization:`